src/pages/guide/language/index.md

```diff
@@ -29,26 +29,24 @@ How Reason Works
 =========================
 
 The OCaml compiler is organized into several stages, which are exposed as
-libraries. `Reason` replaces part of the compiler toolchain with a completely
+libraries. Reason replaces part of the compiler toolchain with a completely
 new syntax parser that is more approachable, while still fully compatible with
-the rest of the compiler.  `Reason` also implements a new source printer which
+the rest of the compiler.  Reason also implements a new source printer which
 integrates into your IDE and the new custom [`REPL`](./tools.html#repl).
 
 ###### Why OCaml?
 
-OCaml is a great tool for writing highly expressive, functional
+OCaml is a great language for writing highly expressive, functional
```

Comment:
"tool" -> "language". Tool isn't necessarily incorrect, but I think this makes it clearer since we typically talk about OCaml as the language and Reason as the tool on top of the language.